en/canary-deployment-tidb-operator.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Canary Deployment of TiDB Operator
+summary: Learn how to canary upgrade TiDB Operator and how to deploy multiple TiDB Operator to manage different TiDB clusters.
+---
+
+# Canary Deployment of TiDB Operator
+
+This document describes how to canary upgrade TiDB Operator and how to deploy multiple TiDB Operator to manage different TiDB clusters.
+
+## Scenarios
+
+- Canary upgrade TiDB Operator.
+- Deploy multiple TiDB Operator to manage different TiDB clusters.
+
+    > **Note:**
+    >
+    > - Currently, you can only deploy multiple `tidb-controller-manager`s and `tidb-scheduler`s.
+    > - Only one Admission Webhook is required.
+    > - If you have deployed multiple TiDB Operator and only some of them enable [Advanced StatefulSet](advanced-statefulset.md), the same TidbCluster Custom Resource (CR) cannot be switched among these TiDB Operator.
+
+## Related parameters
+
+To support canary deployment, the following parameters are added to the `values.yaml` file in the `tidb-operator` chart:
+
+- `appendReleaseSuffix`
+
+    If this parameter is set to `true`, when you deploy TiDB Operator, TiDB Operator adds automatically a prefix (`-{{ .Release.Name }}`) to the name of resources related to `tidb-controller-manager` and `tidb-scheduler`.
```

Comment:
```suggestion
    If this parameter is set to `true`, when you deploy TiDB Operator, TiDB Operator automatically adds a prefix (`-{{ .Release.Name }}`) to the name of resources related to `tidb-controller-manager` and `tidb-scheduler`.
```

Reply:
No, not TiDB Operator, but the helm chart.

Reply:
```suggestion
    If this parameter is set to `true`, when you deploy TiDB Operator, the Helm chart automatically adds a suffix (`-{{ .Release.Name }}`) to the name of resources related to `tidb-controller-manager` and `tidb-scheduler`.
```